libs/GraphicECS/SFML/Resources/SoundResource.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundResources
+*/
+
+#include "SoundResource.hpp"
+
+using namespace graphicECS::SFML::Resources;
+
+void SoundResource::addSound(const SoundResource::soundName_e sound_e, const std::filesystem::path &soundPath)
+{
+    sf::SoundBuffer buffer;
```

Comment:
lock the resource before making any changes

Reply:
don't need to lock in the resource because it's at it creation

Reply:
This function can be called at any moment if you got the resource so you must lock the resource